misc.php

```diff
@@ -79,3 +79,20 @@ function action_wpcom_vip_verify_string() {
 		}
 	}
 }, 10, 1 );
+
+/**
+ * Hooks pre_ping to stop any pinging from happening,
+ * unless `WPCOM_VIP_DO_PINGS` is set to `true` (boolean).
+ *
+ * @param array $post_links The URLs to be pinged
+ */
+function wpcom_vip_pre_ping( $post_links ) {
+	$do_pings = ( defined( 'VIP_DO_PINGS' ) && true === WPCOM_VIP_DO_PINGS );
+	if ( ! $do_pings ) {
```

Comment:
This conditional is still backwards. If we _are_ going to do pings - if `$do_pings` is `true`, then we want to `return` without modifying anything.

If we _aren't_ going to run pings, then we want to clear out the array, _not_ `return`.